app/views/step_by_step_pages/internal_change_notes.html.erb

```diff
@@ -24,26 +24,30 @@
 %>
 
 <%= render 'nav', step_by_step_page: @step_by_step_page, active: "internal-change-notes" %>
+
+<% displayed_edition_number = nil %>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <%= render "govuk_publishing_components/components/accordion", {
-      items: @step_by_step_page.internal_change_notes.map.with_index do |note, index|
-        {
-          heading: {
-            text: "#{note.readable_created_date}"
-          },
-          summary: {
-            text: sanitize("#{note.edition_number ? "Edition #{note.edition_number}" : "Current version"} - Change made by <strong>#{note.author}</strong>")
-          },
-          content: {
-            html: sanitize("
-              <pre class=\"govuk-body change-note--description\">#{note.description}</pre>
-            ")
-          },
-          expanded: index == 0
-        }
-      end
-    } %>
+    <% @step_by_step_page.internal_change_notes.each do |entry| %>
+      <div class="app-c-timeline-entry">
+        <% if displayed_edition_number != entry.edition_number %>
```

Comment:
What happens here if `entry.edition_number` is nil? Originally it displayed `Current version`.

Is the check needed? Will `displayed_edition_number` ever be the same as `entry.edition_number` at this point in the loop?

Reply:
Not anymore, I've removed it in one of the following commits when we changed the approach.

Reply:
Could you remove it from the history?

Reply:
It is needed in this commit and it's doesn't always evaluate to true. It [starts with nil](https://github.com/alphagov/collections-publisher/pull/694/commits/c3105a241af4d66e663f6bc23c33c14c8d74abff#diff-5a189794e316cb715d3507edf2b23b8fR28) then [gets updated every time we reach a new edition](https://github.com/alphagov/collections-publisher/pull/694/commits/c3105a241af4d66e663f6bc23c33c14c8d74abff#diff-5a189794e316cb715d3507edf2b23b8fR52). We're using [a similar technique in content-publisher](https://github.com/alphagov/content-publisher/blob/master/app/views/documents/show/_history_tab.html.erb#L7).

Reply:
I don't get how this working, but I see that it does hide the title, there by "grouping" the change notes. so 👍